docs/source/data_extraction.rst

```diff
@@ -15,12 +15,16 @@ Each Material History corresponds to exactly one row, though the Material Histor
 For example, the material histories of distinct Material Runs will map to exactly two rows even if their histories are identical up to differing Process Runs of their final baking step.
 The only way to define rows right now is through :class:`~citrine.gemtables.rows.MaterialRunByTemplate`, which produces one row per Material Run associated with any of a list of material templates.
 
+The set of tags is optional. The usage scenario is for fine-grained curation. The list of tags should be interpreted as an OR clause, meaning, if a terminal material doesn't contain any of the tags it will be filtered out.
```

Comment:
```suggestion
The set of tags is optional.
The usage scenario is for fine-grained curation.
If a terminal material doesn't contain any of the tags it will be filtered out.
```